### DISCUSSION TITLE
Datatables with search bar?

### DISCUSSION THREAD

**Discussion by SrPeterr** [tabler/tabler#1326] [Q&A]
Hi!
I was just checking https://preview.tabler.io/datatables.html, and I wonder if it is possible to search within the table right now or in the future, thanks! :)

**Comment by oantonioo** [comment]
This template is already too heavy.
Searching inside the table does NOT make sense, especially when a simple javascript fetch could make it faster without having to load all the data in the table.

What I would love for this template to have is an easy way to separate elements and streamline the css and js.
A button to download a LITE template and be able to add elements.
